extensions/panache/hibernate-orm-panache-common/runtime/src/main/java/io/quarkus/hibernate/orm/panache/common/runtime/AbstractJpaOperations.java

```diff
@@ -206,8 +205,10 @@ public PanacheQueryType find(Class<?> entityClass, String query, Sort sort, Obje
                                 + "\" instead");
             }
             NamedQueryUtil.checkNamedQuery(entityClass, namedQuery);
-            return createPanacheQuery(em, query, PanacheJpaUtil.toOrderBy(sort), params);
+            return createPanacheQuery(em, query, null, params);
```

Comment:
Sorting doesn't apply to named queries?

Reply:
No, there is a check a few lines before that throws and exception in case sort is applied to a named query.
We use the query "as is" so we refuse to update it with an ORDER BY clause, people using named query should add the order clause on the named query.

Reply:
Fine, thanks a lot.